temba/channels/tests.py

```diff
@@ -7064,6 +7064,30 @@ def test_send_media(self):
             self.assertEqual(mock.call_args[0][1]['caption'], "Test message")
             self.assertEqual(mock.call_args[0][1]['chat_id'], "1234")
 
+    def test_send_quick_replies(self):
+        metadata = dict(quick_replies=['Yes', 'No'])
+        joe = self.create_contact("Ernie", urn='telegram:1234')
+        msg = joe.send("Test message", self.admin, trigger_send=False, quick_replies=metadata)[0]
+
+        settings.SEND_MESSAGES = True
```

Comment:
Correct pattern for this going forward should instead be:

`@override_settings(SEND_MESSAGES=True)` as a decorator for your method, then ditch the `finally`, etc